tools/archetype/binding/src/main/resources/archetype-resources/README.md

```diff
@@ -0,0 +1,52 @@
+# <bindingName> Binding
+
+_Give some details about what this binding is meant for - a protocol, system, specific device._
+
+_If possible, provide some resources like pictures, a YouTube video, etc. to give an impression of what can be done with this binding._
```

Comment:
Perhaps we should also refer to use the doc sub-folder.
... use for additional resources the *doc* folder ...